README.md

```diff
@@ -69,29 +67,29 @@ Only the `source` import source phase is specified by this proposal.
 
 ### Defining Module Source
 
-Module source is defined through the [ECMA-262 ES modules HostLoadImportedModule refactoring][],
-which moves the construction of module records to the host, and on which we can define a
-`[[ModuleSourceObject]]` custom object to be returned by module source phase that is compatible
-with module expressions.
+The object provided by the module source phase should have a prototype set to the
+`%AbtractModuleSourcePrototype%` object, defined by this specification to be a minimal
```

Comment:
```suggestion
The object provided by the module source phase must be an object that inherits the
`%AbtractModuleSourcePrototype%` object prototypically, defined by this specification to be a minimal
```

Reply:
Or “must be an object with `%AbstractModuleSourcePrototype%` in its chain of prototypes”.